examples/helm/tiller-template.yaml

```diff
@@ -39,6 +40,20 @@ objects:
   - kind: ServiceAccount
     name: tiller
 
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: edit-79929
+    namespace: ${TILLER_NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: edit
+  subjects:
+    - kind: ServiceAccount
+      name: tiller
+      namespace: helm-test-project
```

Comment:
is this value right?  the other rolebinding doesn't set it.

Reply:
Nope this is a typo.